docs/how_to_guides/calculating_elterngeld.ipynb

```diff
@@ -0,0 +1,211 @@
+    "birth of a child. In our case, this would be July 2023 until June 2024. If other income\n",
+    "sources exist, the basis is the average monthly gross income in the calendar year before\n",
+    "the birth of the child, 2023 in our example. We will stick to the first case here\n",
+    "because:\n",
+    "\n",
+    "> **Note**: The Elterngeld implementation in GETTSIM does currently not consider income\n",
+    "> from self-employment or other sources. See\n",
+    "> [this](https://github.com/iza-institute-of-labor-economics/gettsim/issues/613) issue.\n",
+    "\n",
+    "So let's stick to the case where income from dependent employment is the only source of\n",
+    "income. You can compute this directly if you have monthly information on gross income\n",
+    "(e.g. IAB or DRV data). If you have only annual income data, you must choose an\n",
+    "approximation. In most cases, a good approximation is the income of the year before\n",
+    "birth.\n",
+    "\n",
+    "Here, we use made-up data using the `create_synthetic_data` function.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gettsim import (\n",
+    "    compute_taxes_and_transfers,\n",
+    "    create_synthetic_data,\n",
+    "    set_up_policy_environment,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params, functions = set_up_policy_environment(2024)\n",
+    "\n",
+    "data_before_birth = create_synthetic_data(\n",
+    "    n_adults=2,\n",
+    "    n_children=0,\n",
+    "    specs_heterogeneous={\n",
+    "        \"bruttolohn_m\": [[2000.0, 0.0]],\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 2: Approximate net wage before birth\n",
+    "\n",
+    "GETTSIM provides an easy way to compute the relevant net wage\n",
+    "`elterngeld_nettoeinkommen_vorjahr_m` based on step 1 using the target `elterngeld_nettolohn_approximation_m`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "net_wage_approximation = compute_taxes_and_transfers(\n",
+    "    data=data_before_birth,\n",
+    "    functions=functions,\n",
+    "    params=params,\n",
+    "    targets=[\"elterngeld_nettolohn_approximation_m\"],\n",
+    ")\n",
+    "\n",
+    "net_wage_approximation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Compute Elterngeld\n",
+    "\n",
+    "Finally, we can compute Elterngeld using the target `elterngeld_m` and the net wage\n",
+    "information from step 2.\n",
+    "\n",
+    "The reason for not doing this in one GETTSIM call (together with step 2) is that \n",
+    "1. the correct policy date of the net wage calculation (step 2) is always January 1st of\n",
```

Comment:
I thought @JakobWegmann linked a law/directive saying that the parameters throughout the larger part of the year before birth mattered? So for a kid born in March, it would be 2023. For a kid born in November, 2024? Again, I'd argue for keeping our the standard approximation using the previous year, but we should be precise here, also for our own sanity when reading it down the road.

Reply:
One law, three different interpretations 😅 

They write:

>  Die Abzüge für Steuern werden einheitlich für Einkommen aus nichtselbstständiger und selbstständiger Erwerbstätigkeit auf Grundlage einer Berechnung anhand des am 1. Januar des Kalenderjahres vor der Geburt des Kindes für dieses Jahr geltenden Programmablaufplans [...] ermittelt.

My interpretation is that the relevant parameters are the ones of the year before (i.e. child born in December 2024, parameters of January 1st 2023 are considered). I'll adjust the notebook accordingly, but intervene if you understand the law differently.

Reply:
That would be my interpretation of the text, too! @JakobWegmann, could you weigh in one last time?

Reply:
Yes, 1st of January of the calendar year before birth is correct (no matter the birth month).

Reply:
> I thought @JakobWegmann linked a law/directive saying that the parameters throughout the larger part of the year before birth mattered?

For the sake of completeness: That was about the **input values** of the function, so Steuerklasse, place of residence, church tax,...

Reply:
Ahhh. WTF. No wonder they are slow in the Elterngeldstelle.

Reply:
Excellent, if all of you (@MImmesberger @JakobWegmann  and maybe also @pia-molitor ?) could just read through the notebook text and fix any errors in my summary (+maybe add a link to the application form, Jakob?), we'll be good to go!

Reply:
No complaints from my side.

Reply:
Looks very good!

A direct link to a PDF: https://www.zbfs.bayern.de/imperia/md/content/zbfs_intranet/produktgruppe_i/formulare/beeg-antrag/2021_09_anlage_n_mit_info_.pdf

I find this part a bit confusing:
> The Lohnsteuerklasse is always set on a yearly basis; if it changes with the start of the calendar year a child is born, the date of birth matters. Up to June, the Lohnsteuerklasse from the previous calendar year will be relevant; for July or later, the Lohnsteuerklasse from the current calendar year will be relevant.

I would simply delete it. Eventually, also the Steuerklasse is a choice of the data preparer.

Reply:
Depends on the dataset, I guess 😄

I don't care either way, your call @mimmesberger, and then go ahead and merge!

Reply:
I left it in there because it helped me to understand the purpose of the paragraph. I just added a "for example" to make clear that this reasoning also applies to the other variables.